### PR TITLE
Add timeline section

### PR DESCRIPTION
**Pull request by LucasLobo**
I added a timeline section. It is easily customizable by changing the values in the `variables.scss` file. Content can be added or changed in the `timeline.yml` file.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -3,6 +3,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Swift Megaminds</title>
     <link lang='css' rel="stylesheet" href="{{ site.baseurl }}/assets/css/main.css">
+    <script
```
Comment (sachsom95) [inline _includes/header.html:6]: Minor nit picks is the linting consistent for script tag here?

Comment (sachsom95) [reply _includes/header.html:6]: just comment if its the style being followed just to make sure and I'll approve

Comment (LucasLobo) [reply _includes/header.html:6]: I honestly don't know. I can probably condense it in one line.

Comment (sachsom95) [reply _includes/header.html:6]: i think multi line is better, keep it for better readability just that others seemed to have kept it in one line but i like this better

File: _sass/all.scss [hunk 2]
```diff
@@ -1,14 +1,22 @@
```
Comment (sachsom95) [inline _sass/all.scss]: could you remove the extra line at top

File: _sass/timeline.scss [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+}
+
+.cd-horizontal-timeline a {
+  color: $timeline-accent-color;
+  text-decoration: none;
+}
+
+/* -------------------------------- 
+
+Main Components
```
Comment (sachsom95) [inline _sass/timeline.scss]: Minor nit picks maybe remove the additional lines here

File: _sass/variables.scss [hunk 4]
```diff
@@ -1,8 +1,27 @@
...
 $hero_font:  'Space Mono', monospace;
-$font_family: "Open Sans", sans-serif;
+
 $var_color: #F87669;
 $var_name_color:  #0ea87f;
 $var-value_color: #fcb350;
 $title_color: white;
 $wrap_color: #EED0D4;
+
+
```
Comment (sachsom95) [inline _sass/variables.scss]: since these are all variables we can just leave a single line of space for different groups

**Review by sachsom95 — APPROVED** [review]
All good Nice work :)

**Comment by sachsom95** [comment]
Merge 2 underway local tests work

**Pull request merged**